src/server/acl/acl_family.cc

```diff
@@ -48,12 +49,22 @@ void AclFamily::Acl(CmdArgList args, ConnectionContext* cntx) {
 void AclFamily::List(CmdArgList args, ConnectionContext* cntx) {
   const auto registry_with_lock = ServerState::tlocal()->user_registry->GetRegistryWithLock();
   const auto& registry = registry_with_lock.registry;
-
   (*cntx)->StartArray(registry.size());
+
+  auto pretty_print_sha = [](std::string_view pass) {
+    std::string result;
+    for (size_t i = 0; i < 15; ++i) {
+      absl::StrAppend(&result, absl::Hex(pass[i]));
+    }
+
+    result.resize(15);
```

Comment:
1. What about the leading digit? [1, 1] should be `0101`

I thinks its better to use what Roy found 😝 

```

// BytesToHexString()
//
// Converts binary data into an ASCII text string, returning a string of size
// `2*from.size()`.
std::string BytesToHexString(absl::string_view from);
```

Reply:
100%